src/Microsoft.Health.Fhir.Liquid.Converter/Filters/CollectionFilters.cs

```diff
@@ -65,23 +65,13 @@ public static object GetIndex(object[] collection, int index)
             return null;
         }
 
-        public static object Slice(object input, int s, int n = 1)
+        public static object SliceArray(object[] input, int s, int n = 1)
```

Comment:
Not sure if the standard filters are available but it does look like there is a function for this built into dotliquid? https://github.com/dotliquid/dotliquid/blob/c4d7744a4b6f9669209ebeeee099ffff4f7da8ee/src/DotLiquid/StandardFilters.cs#L76